docset/winserver2022-ps/failoverclusters/New-Cluster.md

```diff
@@ -104,25 +112,26 @@ Name
 cluster1
 ```
 
-This example creates a four-node cluster named cluster1.
-The cluster uses the static IP address 2.0.0.123, and does not use the network 3.0.0.0/8.
+This example creates a four-node cluster named cluster1. The cluster uses the static IP address
+2.0.0.123, and does not use the network 3.0.0.0/8.
 
 ## PARAMETERS
 
 ### -AdministrativeAccessPoint
-Specifies the type of administrative access point that the cmdlet creates for the cluster.
-The acceptable values for this parameter are:
-
-- ActiveDirectoryAndDns.
-The cmdlet creates an administrative access point for the cluster.
-The administrative access point is registered in DNS and enabled in Active Directory Domain Services.
-- Dns.
-The cmdlet creates an administrative access point for the cluster.
-The administrative access point is registered in DNS but is not enabled in Active Directory Domain Services.
+
+Specifies the type of administrative access point that the cmdlet creates for the cluster. The
+acceptable values for this parameter are:
+
+- ActiveDirectoryAndDns. The cmdlet creates an administrative access point for the cluster. The
+  administrative access point is registered in DNS and enabled in Active Directory Domain Services.
+- Dns. The cmdlet creates an administrative access point for the cluster. The administrative access
+  point is registered in DNS but is not enabled in Active Directory Domain Services.
```

Comment:
To keep the in-console help from breaking, we need blank lines between list items in parameters.

```suggestion
- ActiveDirectoryAndDns. The cmdlet creates an administrative access point for the cluster. The
  administrative access point is registered in DNS and enabled in Active Directory Domain Services.

- Dns. The cmdlet creates an administrative access point for the cluster. The administrative access
  point is registered in DNS but is not enabled in Active Directory Domain Services.

```